cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/io/cdap/cdap/datapipeline/LineageOperationProcessorTest.java

```diff
@@ -56,6 +57,8 @@ public void testSimplePipeline() {
     List<FieldOperation> fieldOperations = new ArrayList<>();
     fieldOperations.add(new FieldReadOperation("read", "reading data", EndPoint.of("default", "file"), "offset",
                                                "body"));
+    fieldOperations.add(new FieldTransformOperation("transform", "transoform in src",
```

Comment:
typo: transoform -> transform

Reply:
This is added to test some scenario and it broke the test, reverted the change